src/figuresClasses.ts

```diff
@@ -1,19 +1,89 @@
-export interface Figure {
+enum Color {
+  Blue = 'blue',
+  Red = 'red',
+  Green = 'green',
+}
+
+enum Shape {
+  Rectangle = 'rectangle',
+  Circle = 'circle',
+  Triangle = 'triangle',
+}
 
+export interface Figure {
+  shape: Shape,
+  color: Color,
+  getArea(): number,
 }
 
-export class Triangle {
+function roundVolue(volue: number): number {
+  return Math.floor(volue * 100) / 100;
+}
 
+function isCorrectSides(...sides: number[]): void {
```

Comment:
This naming is good for variable but for function name should means action. Ex. `check...` `get..` `calculate...` 
https://medium.com/swlh/how-to-better-name-your-functions-and-variables-e962a4ef335b